src/Microsoft.Health.Dicom.Core/Features/ExtendedQueryTag/ExtendedQueryTagEntry.cs

```diff
@@ -26,20 +26,9 @@ public class ExtendedQueryTagEntry
         /// </summary>
         public string PrivateCreator { get; set; }
 
-        /// <summary>
-        /// Level of this tag. Could be Study, Series or Instance.
-        /// </summary>
-        public QueryTagLevel Level { get; set; }
-
-        /// <summary>
-        /// Status of this tag. Represents the current state the tag is in.
-        /// This value is null when the entry represents a tag to be created.
-        /// </summary>
-        public ExtendedQueryTagStatus Status { get; set; }
-
         public override string ToString()
```

Comment:
By the way, what is the purpose of this `ToString()`? Is it for error messages?

Reply:
Yup!